biz.aQute.bnd.annotation/src/aQute/bnd/annotation/spi/Constants.java

```diff
@@ -15,6 +15,8 @@ private Constants() {}
 
 	public static final String	REGISTER_MACRO			= "register:=${#register}";
 
+	public static final String	SERVICE_MACRO			= "objectClass:List<String>=${#value}";
```

Comment:
Shouldn't you quote the value since it is a `List`?

`"objectClass:List<String>=\"${#value}\""`

Reply:
will do!